<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="GSrilekha" failures="2" tests="6" name="Test" time="86.694" errors="0" timestamp="2025-03-05T16:49:02 IST">
  <testcase classname="TestCases.LoginDataDrivenTest" name="DDTLoginTest" time="15.327">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at TestCases.LoginDataDrivenTest.DDTLoginTest(LoginDataDrivenTest.java:30)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- DDTLoginTest -->
  <testcase classname="TestCases.LoginDataDrivenTest" name="DDTLoginTest" time="15.217"/>
  <testcase classname="TestCases.LoginDataDrivenTest" name="DDTLoginTest" time="15.896"/>
  <testcase classname="TestCases.LoginDataDrivenTest" name="DDTLoginTest" time="7.248"/>
  <testcase classname="TestCases.LoginDataDrivenTest" name="DDTLoginTest" time="15.709"/>
  <testcase classname="TestCases.LoginDataDrivenTest" name="DDTLoginTest" time="7.73">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at TestCases.LoginDataDrivenTest.DDTLoginTest(LoginDataDrivenTest.java:37)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- DDTLoginTest -->
</testsuite> <!-- Test -->
